src/types/api.js

```diff
@@ -73,6 +73,34 @@
  * @prop {boolean} [$anchorTimeout]
  */
 
+/**
+ * @typedef AnnotationData
```

Comment:
I assume this is meant to be a typedef for the object that is exchanged between the sidebar and annotator to represent an annotation?

I would suggest to put that in a separate file (`types/annotator.js`?) since it isn't something that comes from/is sent to the hypothesis API directly.